app/main.py

```diff
@@ -2,0 +1,44 @@
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
```

Comment:
You are using `append()` method here to add a single value to the `skills` list. According to the checklist, you should use `extend()` method or `+=` operator even when adding a single value to a list. Please, correct this.